src/FluentDriver.php

```diff
@@ -79,7 +102,74 @@ public function isTransient($className)
     }
 
     /**
+     * @param array $paths
+     *
+     * @throws MappingException
+     */
+    public function loadPaths(array $paths)
+    {
+        $classes = [];
+        $includedFiles = [];
+
+        foreach ($paths as $path) {
+            if (!is_dir($path)) {
+                throw MappingException::fileMappingDriversRequireConfiguredDirectoryPath($path);
+            }
+
+            $iterator = new RegexIterator(
+                new RecursiveIteratorIterator(
+                    new RecursiveDirectoryIterator($path, FilesystemIterator::SKIP_DOTS),
+                    RecursiveIteratorIterator::LEAVES_ONLY
+                ),
+                '/^.+'.preg_quote($this->fileExtension).'$/i',
+                RecursiveRegexIterator::GET_MATCH
+            );
+
+            foreach ($iterator as $file) {
+                $sourceFile = $file[0];
+
+                if (!preg_match('(^phar:)i', $sourceFile)) {
+                    $sourceFile = realpath($sourceFile);
+                }
+
+                require_once $sourceFile;
```

Comment:
@patrickbrouwers I don't like this strategy, because it depends on those files not being previously loaded. I don't know what would happen with opcache, for example.
I think we talked about this in the previous PR.
I'd agree to merge as-is, but I wouldn't trust this strategy for my projects.